src/main/java/com/zontik/groshiky/dao/UserDao.java

```diff
@@ -15,8 +18,17 @@
     @Autowired
     private SessionFactory sessionFactory;
 
+    @Autowired
```

Comment:
Please use a constructor for autowiring as @RomanKashenok  already has asked you :)